symplyphysics/laws/dynamics/vector/centrifugal_force_via_centripetal_acceleration.py

```diff
@@ -0,0 +1,52 @@
+from symplyphysics import (
+    units,
+    Quantity,
+    QuantityVector,
+    Vector,
+    validate_input,
+    validate_output,
+    symbols,
+    scale_vector,
+    vector_magnitude,
+)
+
+# Description
+## The centrifugal force is a fictitious force that only exists in a non-inertial coordinate system.
+## It disappears entirely when transitioning into an inertial frame of reference. It appears to act
+## on all objects when viewed in a rotating frame of reference and is directed away from the axis
+## of rotation.
+
+# Law: F_cf = -m * a_cp
```

Comment:
What about the Coriolis force? It would also have a minus sign in the formula. So it's not really the `F = ma` law because these forces are fictitious.

Reply:
I see. I missed that you use centripetal acceleration here. Isn't it better to define centrifugal acceleration as minus centripetal acceleration?

Reply:
I can do that. The Coriolis force $`\vec F_\text{cor} = - m \vec a_\text{cor}`$ still requires a separate law though.

Reply:
No, it's still `F = ma`

Reply:
But it doesn't account for the minus sign. The Coriolis acceleration and the Coriolis force are oppisitely directed by definition.

Reply:
I disagree. Force is what makes acceleration. They should have the same sign. It seems we are talking about different accelerations.

Reply:
I guess that is a confusing naming, but it is a convention. Basically, we have [the relative motion law](https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/dynamics/vector/relative_acceleration_from_force.py), which is equivalent to $`m \vec a_\text{rel} = \vec F - m \vec a_\text{cor} - m \vec a_\text{tr}`$. It is conventional and quite logical to re-write that as $`m \vec a_\text{rel} = \vec F + \vec F_\text{cor} + \vec F_\text{cf} + \dots`$ So the minus sign is included in the those forces.

Reply:
Force may have minus sign. But it is not logical to have inverse force-acceleration relation. It is acceptable only if there is commonly used acceleration and we want to link it to indirectly dependent force. Actually, coriolis acceleration is rarely used so we may decide to avoid using it at all.

Reply:
Well it seems to be a matter of convention ultimately. Some sources follow the way I presented it above, while others include the minus sign in the corresponding accelerations. I agree that the minus should probably go into the formula for accelerations, that would just require a revision of already implemented laws.

By the way, the Coriolis force is used extensively in fluid mechanics and atmospheric physics, to say the least, so I don't think we should just avoid it.

Reply:
I said about coriolis acceleration. While coriolis force is actually being used, acceleration is rarely mentioned.